src/CsvHelper/CsvReader.cs

```diff
@@ -88,7 +88,7 @@ public class CsvReader : IReader
 		/// <param name="parser">The <see cref="IParser" /> used to parse the CSV file.</param>
 		public CsvReader(IParser parser)
 		{
-			Configuration = parser.Configuration as IReaderConfiguration ?? throw new ConfigurationException($"The {nameof(IParser)} configuration must implement {nameof(IReaderConfiguration)} to be used in {nameof(CsvReader)}.");
+			Configuration = parser.Configuration as IReaderConfiguration ?? throw new CsvHelper.Configuration.ConfigurationException($"The {nameof(IParser)} configuration must implement {nameof(IReaderConfiguration)} to be used in {nameof(CsvReader)}.");

```

Comment:
I think `CsvHelper.Configuration` may already be imported. If it is then the change on this file is not needed, surely, and hence one less file for Josh to review?